compose/ui/ui/src/uikitMain/kotlin/androidx/compose/ui/uikit/ComposeUIViewControllerConfiguration.uikit.kt

```diff
@@ -103,15 +104,11 @@ sealed interface OnFocusBehavior {
     /**
      * The Compose view will stay on the current position.
      */
-    object DoNothing : OnFocusBehavior
+    data object DoNothing : OnFocusBehavior
```

Comment:
[Avoid using data classes in your API﻿](https://kotlinlang.org/docs/api-guidelines-backward-compatibility.html#avoid-using-data-classes-in-your-api)

Reply:
These are data objects, not classes, so they cannot be changed in the way it described in the article.